variables.tf

```diff
@@ -217,6 +217,12 @@ variable "execution_role_arn" {
   default     = null
 }
 
+variable "drain_container_definitions" {
```

Comment:
Do you think it would be useful to create two vars here:

- `drain_container_definitions` - so exactly like you have.
- `additional_drain_container_definitions` - this would allow people to add sidecars without having to also replace the actual drain container.

The two would be mutually exclusive, but the second option might make sidecars simpler.

Reply:
yeah, that's what I wanted to avoid with this. :D specifically, because of the dependencies where the main container would depend on the sidecar. that wouldn't be possible with `additional_drain_container_definitions`. I thought providing the entire thing is just easier.

Reply:
Ok - fair enough.